frontend/src/action-creator.async.js

```diff
@@ -63,7 +63,8 @@ import {
   USER_PUBLIC_PROFILE,
   FAVORITE_LIST,
   FAVORITE,
-  UNREAD_NOTIFICATION_COUNT
+  UNREAD_NOTIFICATION_COUNT,
+  ROLE_WORKSPACE_LIST
```

Comment:
same remark about naming: is this a list of workspaces?

Reply:
yes, it's a list of workspace with the role of the user for each of those workspace

Reply:
Yes I saw this; this is a good opportunity to improve it

Reply:
So it's a list of roles (each coming with its own workspace), not a list of workspaces.

To make this more clear it should be renamed to WORKSPACE_ROLE_LIST or better MEMBERSHIP_LIST. Actually, I expect you add a TODO for future renaming instead of refactor right now (according to our private discussion: https://algoo.tracim.fr/ui/workspaces/165/contents/thread/134421)

Reply:
why not USER_ROLE_WORKSPACE to match the backend naming?
the refactoring in frontend is fairly simple and can be done right away

Reply:
as seen together, we have only rename the endpoint and the we'll rename the code in later issue